packages/@aws-cdk/aws-ecs-patterns/lib/base/scheduled-task-base.ts

```diff
@@ -1,7 +1,6 @@
-import { Schedule } from '@aws-cdk/aws-applicationautoscaling';
 import { IVpc, SubnetSelection, SubnetType } from '@aws-cdk/aws-ec2';
 import { AwsLogDriver, Cluster, ContainerImage, ICluster, LogDriver, Secret, TaskDefinition } from '@aws-cdk/aws-ecs';
-import { Rule } from '@aws-cdk/aws-events';
+import { Rule, Schedule } from '@aws-cdk/aws-events';
```

Comment:
Did we just move Schedule to another module or is this a different implementation?

Reply:
There are at least three almost identical implementations in events, autoscaling and applicationautoscaling.

Reply:
@piradeepk - looks like the dependency pulled in Schedule from `autoscaling` instead of `events` in [this PR](https://github.com/aws/aws-cdk/pull/3404/files#diff-3b0296cb1bd5e0de44799743646c98683475e9af38c57aacaa7292ecb59bd7d5)

Reply:
Got it! Thanks @shivlaks